tests/Fixtures/DoctrineType/ValueObjectWithToStringImpl.php

```diff
@@ -15,20 +15,23 @@
 
 final class ValueObjectWithToStringImpl
```

Comment:
now looking at this one, is similar to the previous one, I don't think `$uuid` was meant to be a string.